packages/terra-clinical-item-collection/src/ItemCollection.jsx

```diff
@@ -93,6 +93,7 @@ const ItemCollection = (props) => {
     ...customProps
   } = props;
 
+  const ariaLabel = customProps['aria-label'];
```

Comment:
Should we make this a first-class prop?

Reply:
this is spreading on the div wrapping the responsive element below. Should it be removed? Should it spread on the table?

Reply:
Added ariaLabel as a first-class prop. The label is no longer being spread on the wrapping div and also passed down to table. 
https://github.com/cerner/terra-clinical/pull/644/commits/75438d43ea6e9d600bde3846d966c3082372e338